### PR TITLE
feat(parsers): add scheme parser

### PR DESCRIPTION
**Pull request by steelsojka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,6 +296,7 @@ List of currently supported languages:
 - [ ] jsdoc
 - [x] dart (maintained by @Akin909)
 - [x] rst (maintained by @stsewd)
+- [x] scheme (maintained by @steelsojka)
```
Comment (theHamsta) [inline README.md:299]: Tree-sitter query language? It's not real scheme.

Comment (steelsojka) [reply README.md:299]: I wasn't sure exactly. What should the official "language" name be? ts-query?

Comment (steelsojka) [reply README.md:299]: I'll rename the parser as well...

Comment (steelsojka) [reply README.md:299]: Does the filetype need to change? `.scm` files are detected as 'scheme' (at least with my setup)

Comment (theHamsta) [reply README.md:299]: Should be enough not make that clear in the README.md. I guess you can keep all other usages of scheme. Until someone wants to parse real scheme... :smile:

**Pull request merged**